### ISSUE TITLE
Submodule aiofiles.os not recognized

### ISSUE THREAD

**Issue by MKuranowski** [microsoft/pyright#1123]
**Describe the bug**
Pylance doesn't recognize the `aiofiles.os` submodule. A `"os" is not a known member of module` error is produced when trying to use functionality from aiofiles.os.

**Expected behavior**
Since typeshed includes signatures of the aiofiles.os submodule (<https://github.com/python/typeshed/blob/master/third_party/3/aiofiles/_os.pyi>), using anything from aiofiles.os should be properly type-checked and shouldn't be reported as an error

**Screenshots or Code**
```python
import aiofiles

async def remove(x: str):
    await aiofiles.os.remove(x)

```
![image](https://user-images.githubusercontent.com/16469272/97105984-39d8b480-16bf-11eb-9a4a-d358ea50ca4d.png)

**VS Code extension or command-line**
VS Code extension: pylance version 2020.10.2.

**Comment by erictraut** [comment]
According to the [aiofiles type stub in typeshed](https://github.com/python/typeshed/blob/master/third_party/3/aiofiles/__init__.pyi), the `aiofiles` top-level module doesn't import and re-export the `os` submodule. So pyright is doing the correct thing in indicating that you cannot import `aiofiles` and then access `aiofiles.io`. If you change your code to `import aiofiles.os`, it will work.

I've looked at the source code for `aiofiles`, and the top-level module does appear to re-export the submodule, so this appears to be a bug in the typeshed stub. You're welcome to submit a bug report or a PR to the typeshed repo. Pyright regularly picks up changes from that repo.
